### PR TITLE
[PGPNP] Add basic SSL support to signer and other improvements

### LINKED ISSUE #4135: Add SSL support to Signer

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Add basic SSL support, configurable via env vars
- Add automatic request logging middleware Morgan
- Improve logging format
- Normalize script file names and do some cleanup

Screen Shot 2020-06-17 at 19 05 38

### Tested

Locally

### Related issues

- Fixes #4135 

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/phone-number-privacy/combiner/scripts/run-migrations.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// tslint:disable: no-console
```
Comment (jmrossy) [inline packages/phone-number-privacy/combiner/scripts/run-migrations.ts:1]: Mostly just renamed the file and fixed the hanging-after-migration issue

File: packages/phone-number-privacy/signer/src/index.ts [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 async function start() {
+  logger.info(`Starting. Dev mode: ${DEV_MODE}`)
   await initKeyProvider()
 
-  logger.info(`Starting server. Dev mode: ${DEV_MODE}`)
+  const server = createServer()
+  logger.info(`Starting server.`)
   const port = config.server.port
-  app.listen(port, () => {
+  server.listen(port, () => {
```
Comment (codyborn) [inline packages/phone-number-privacy/signer/src/index.ts:13]: should this be `createServer`?

**Comment by jmrossy** [comment]
I'm gonna add an option to configure the Google service account path, per Peter's feedback

**Review by codyborn — APPROVED** [review]
LGTM! We can also add some E2E tests to verify http traffic is dropped if https is enabled.

**Review by jcortejoso — APPROVED** [review]
:partying_face:

**Pull request merged**